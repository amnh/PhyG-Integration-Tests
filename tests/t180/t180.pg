--build(character, all-data), inputting Hennig86 files with all character settings included
set(seed:1634561640)
read(rename: "woman_rename.txt")
--woman_allchars.ss includes cc+, cc-, continuous, ?, [ ], /N 
read(tnt: "woman_allchars.ss")
read(tnt: "woman_step.ss")
read(include:"Include_woman.txt")
build(character, replicates:3)
report("t180.csv", data, overwrite)
report("t180.tre", graphs, newick, graphs, overwrite)
