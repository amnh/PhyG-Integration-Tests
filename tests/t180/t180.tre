(AlbanianTosk:0.0,((Irish:2.7,(Icelandic:63.0,(KolschColognian:6.0,(EasternFrisianSaterland:9.2,(Yiddish:40.3,((Assamese:34.6,SanskritVedic:14.1)HTU27:17.0,(Danish:24.9,(AncientIonicGreek:13.3,(Scots:19.0,(Faroese:23.0,(ArmenianWestern:24.0,Cornish:17.0)HTU30:81.7)HTU22:15.0)HTU26:29.3)HTU19:24.0)HTU29:23.0)HTU17:13.0)HTU21:3.6)HTU20:38.0)HTU24:11.3)HTU18:5.9)HTU23:41.9,(Manx:6.1,OldPrussian:7.4)HTU28:32.0)HTU25:0.0)HTU16[1151.3000000000002];
(AlbanianTosk:0.0,(OldPrussian:28.3,(Manx:4.1,((Irish:2.5,(EasternFrisianSaterland:6.0,((Icelandic:73.0,KolschColognian:8.3)HTU18:34.0,((AncientIonicGreek:10.2,(Danish:34.0,(Yiddish:39.0,(Assamese:42.5,SanskritVedic:2.2)HTU25:18.0)HTU26:22.2)HTU29:16.0)HTU21:5.0,(Faroese:29.0,Scots:8.0)HTU23:45.4)HTU19:21.0)HTU17:4.9)HTU27:4.2)HTU20:56.0,(ArmenianWestern:13.0,Cornish:27.0)HTU24:43.4)HTU22:12.1)HTU28:32.1)HTU30:0.0)HTU16[1134.3000000000002];
(AlbanianTosk:0.0,((Irish:1.5,(((((AncientIonicGreek:26.2,Danish:36.0)HTU18:11.0,(Yiddish:29.0,(Assamese:42.5,SanskritVedic:4.2)HTU29:17.0)HTU27:12.2)HTU20:4.0,(EasternFrisianSaterland:7.9,(Icelandic:74.0,KolschColognian:7.3)HTU28:23.0)HTU23:14.0)HTU22:36.2,(Faroese:27.0,Scots:10.0)HTU24:32.2)HTU17:50.5,(ArmenianWestern:13.0,Cornish:27.0)HTU30:43.0)HTU19:1.4)HTU21:23.1,(Manx:6.1,OldPrussian:7.4)HTU26:41.0)HTU25:0.0)HTU16[1113.7];
