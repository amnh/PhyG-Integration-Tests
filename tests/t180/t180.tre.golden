(AlbanianTosk:97.4,((Cornish:193.20000000000002,(Yiddish:85.64999999999999,(EasternFrisianSaterland:90.45,(Faroese:71.0,(Icelandic:83.1,(AncientIonicGreek:60.4,Assamese:41.7)HTU17:31.95)HTU18:45.1)HTU19:9.1)HTU20:22.2)HTU21:10.1)HTU22:44.9,(KolschColognian:31.7,(OldPrussian:119.0,(Scots:97.5,(SanskritVedic:40.55,(Manx:185.70000000000002,(Danish:80.05,(ArmenianWestern:35.95,Irish:201.1)HTU30:105.19999999999999)HTU29:6.6)HTU28:43.35)HTU27:16.6)HTU26:41.1)HTU25:36.7)HTU24:23.6)HTU23:79.9)HTU16[1681.3999999999999];
(AlbanianTosk:112.15,((Danish:37.85,(AncientIonicGreek:24.299999999999997,(Scots:84.25,Yiddish:99.6)HTU20:47.8)HTU18:25.299999999999997)HTU17:12.45,(ArmenianWestern:148.55,(KolschColognian:69.6,(EasternFrisianSaterland:100.1,(Faroese:78.85,(OldPrussian:92.35,(Manx:190.29999999999998,(SanskritVedic:27.05,(Icelandic:111.1,(Assamese:29.05,(Cornish:160.1,Irish:204.45)HTU30:50.3)HTU29:39.6)HTU28:20.6)HTU27:28.25)HTU26:47.1)HTU25:43.9)HTU24:21.1)HTU23:25.75)HTU22:39.699999999999996)HTU21:33.6)HTU19:85.65)HTU16[1673.4000000000003];
(AlbanianTosk:84.0,(Irish:195.29999999999998,(OldPrussian:89.9,(EasternFrisianSaterland:107.55,(Yiddish:112.30000000000001,(SanskritVedic:43.6,(ArmenianWestern:109.35000000000001,(Faroese:48.1,(Cornish:191.29999999999998,(AncientIonicGreek:21.1,((Assamese:52.7,(Icelandic:83.1,(KolschColognian:24.4,Scots:130.0)HTU17:19.1)HTU18:41.95)HTU19:38.1,(Danish:31.1,Manx:211.65)HTU29:30.95)HTU20:30.4)HTU21:24.650000000000002)HTU22:32.6)HTU23:29.6)HTU24:60.35)HTU25:9.6)HTU26:8.45)HTU27:69.9)HTU28:18.55)HTU30:54.0)HTU16[1466.2999999999997];
